JASP-Common/utils.cpp

```diff
@@ -399,3 +401,36 @@ std::string Utils::doubleToString(double dbl)
 	conv << dbl;
 	return conv.str();
 }
+
+// hex should be 4 hexadecimals characters
+std::string Utils::_convertEscapedUnicodeToUTF8(const std::string& hex)
+{
+	std::string result = hex;
+	std::istringstream iss(hex);
+
+	uint32_t bytes;
+	std::wstring_convert<std::codecvt_utf8<char32_t>, char32_t> conv;
```

Comment:
Could be `static const` maybe? you are reusing it quite a lot.

Reply:
This function is used only exceptionally, only when a gettext returns a <U+FFFF> string. But a static is not wrong.